src/components/routes/homepage/EventTimePicker.js

```diff
@@ -1,25 +1,35 @@
 import React from 'react';
 import { makeStyles } from '@material-ui/core';
 import { MuiPickersUtilsProvider, DateTimePicker } from '@material-ui/pickers';
+import { ThemeProvider, useTheme } from '@material-ui/core/styles';
 import MomentUtils from '@date-io/moment';
 
 const useStyles = makeStyles(theme => ({
-  timepicker: {
+  timePicker: {
     width: '100%'
   }
 }));
 
-export const EventTimePicker = ({ value, setValue }) => {
+export const EventTimePicker = ({ value, handleTimeChange }) => {
   const classes = useStyles();
 
+  const theme = useTheme();
+  const { overrides } = theme;
+  const timePickerTheme = {
+    ...theme,
+    overrides: { ...overrides, MuiButton: {} }
+  }; // removes MuiButton override styling
```

Comment:
@welvin21 I changed the approach because `overrides` object may contain potential components in the future that we want to pass down

Reply:
@98sean98 Ohh yeah, that's true, thanks for pointing out 👍